test/commands/list-kinesis-streams.test.js

```diff
@@ -0,0 +1,108 @@
+const { expect, test } = require("@oclif/test");
```

Comment:
the indentation in this file seems completely different to the rest of the project for some reason, is it using tabs instead of spaces?

Reply:
@theburningmonk I think so, that's what prettier gave me, do you want me to change it?